spec/presenters/content_store_presenter_spec.rb

```diff
@@ -0,0 +1,121 @@
+        need_ids: ["100123", "100124"],
+        phase: "beta",
+        public_updated_at: "2014-05-14T13:00:06Z",
+        publishing_app: "mainstream_publisher",
+        redirects: [],
+        rendering_app: "mainstream_frontend",
+        routes: [{:path=>"/vat-rates", :type=>"exact"}],
+        title: "VAT rates",
+        transmitted_at: Time.new.to_f,
+      )
+    end
+  end
+
+  context "for a draft content item" do
+    let!(:content_item) { FactoryGirl.create(:draft_content_item) }
+    let!(:link_set) { FactoryGirl.create(:link_set, content_id: content_item.content_id) }
+
+    it "presents the object graph for the content store" do
+      result = described_class.present(content_item)
+
+      expect(result).to eq(
+        content_id: content_item.content_id,
+        base_path: "/vat-rates",
+        access_limited: nil,
+        analytics_identifier: "GDS01",
+        description: "VAT rates for goods and services",
+        details: {:body=>"<p>Something about VAT</p>\n"},
+        format: "guide",
+        links: link_set.links,
+        locale: "en",
+        need_ids: ["100123", "100124"],
+        phase: "beta",
+        public_updated_at: "2014-05-14T13:00:06Z",
+        publishing_app: "mainstream_publisher",
+        redirects: [],
+        rendering_app: "mainstream_frontend",
+        routes: [{:path=>"/vat-rates", :type=>"exact"}],
+        title: "VAT rates",
+        transmitted_at: Time.new.to_f,
+      )
+    end
+  end
+
+  describe described_class::V1 do
+    let(:attributes) do
+      {
+        content_id: "content_id",
+        access_limited: "access_limited",
+        update_type: "update_type",
+      }
+    end
+
+    around do |example|
+      Timecop.freeze { example.run }
+    end
+
+    it "presents all attributes by default and mixes in transmitted_at" do
+      result = described_class.present(attributes)
+
+      expect(result).to eq(
+        content_id: "content_id",
+        access_limited: "access_limited",
+        update_type: "update_type",
+        transmitted_at: Time.new.to_f,
+      )
+    end
+
+    it "can optionally remove the access_limited attribute" do
+      result = described_class.present(attributes, access_limited: false)
+
+      expect(result).to eq(
+        content_id: "content_id",
+        update_type: "update_type",
+        transmitted_at: Time.new.to_f,
+      )
+    end
+
+    it "can optionally remove the update_type attribute" do
+      result = described_class.present(attributes, update_type: false)
+
+      expect(result).to eq(
+        content_id: "content_id",
+        access_limited: "access_limited",
+        transmitted_at: Time.new.to_f,
+      )
+    end
+
+    it "can optionally omit the transmitted_at" do
+      result = described_class.present(attributes, transmitted_at: false)
+
+      expect(result).to eq(
+        content_id: "content_id",
+        access_limited: "access_limited",
+        update_type: "update_type",
+      )
+    end
+  end
+end
```

Comment:
Would be good to break out these tests so they test distinct presenters for the purpose.  Also there's a lot of uses of `:something => "a thing"` where `something: "a thing"` would do.